jenkins/packer/llm-studio-aws.json

```diff
@@ -17,8 +17,32 @@
         "ssh_username": "ubuntu",
         "tags": 
           {
-            "name": "h2oai-llm-studio-{{user `llm_studio_version`}}"
+            "name": "h2oai-llm-studio-{{user `llm_studio_version`}}",
+            "Owner": "ops@h2o.ai",
+            "Project": "DevOps",
+            "Department": "Engineering",
+            "Environment": "Dev",
+            "Scheduling": "self-managed"
           },
+
+          "run_volume_tags": {
+            "Name": "H2oai-dai-{{user `dai_version`}}",
```

Comment:
dai?

Reply:
This is fixed now @pascal-pfeiffer 
Thank you!